cmd/init_project.go

```diff
@@ -68,21 +68,23 @@ kubebuilder init --domain example.org --license apache2 --owner "The Kubernetes
 }
 
 type projectOptions struct {
-
-	// flags
-	fetchDeps          bool
-	skipGoVersionCheck bool
-
 	boilerplate project.Boilerplate
 	project     project.Project
 
-	// deprecated flags
-	dep     bool
+	// final result
+	scaffolder scaffold.ProjectScaffolder
+
+	// @Deprecated
 	depFlag *flag.Flag
+	// @Deprecated
 	depArgs []string
 
-	// final result
-	scaffolder scaffold.ProjectScaffolder
+	// flags
+	fetchDeps          bool
+	skipGoVersionCheck bool
+
+	// @Deprecated
```

Comment:
> @Deprecated

Is it required to use this format?
Can you please point me to the document if any?

Reply:
No. The check has no relation with the comment.
I will replace for the previous comment since shows that you prefer.

Reply:
Done.